docs/telemetry.md

```diff
@@ -57,10 +65,10 @@ Property | Description
 `user.storeRegion` |
 
 
-### User Session telemetry
+#### User Session telemetry
 
 A session represents a series of user actions. A session starts with a user action.
-It ends at the last user activity when there is no more activity for 30 minutes, or if it lasts longer than 30 minutes.
+It ends at the last user action when there's been no further activity for 30 minutes, or if it lasts longer than 30 minutes.
```

Comment:
I don't understand "...or if it lasts longer than 30 minutes.". Do you count just 30 minutes or 30 minutes plus the activity delta?

Reply:
Sessions expire on 30minutes of inactivity or after 24hours. The prior wordings seems just wrong.